examples/apps/ai-collab/src/app/tasks-list/presence.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	IPresence,
+	LatestMap,
+	type PresenceStates,
+	type PresenceStatesSchema,
+} from "@fluid-experimental/presence";
+
+export interface User {
+	photo: string;
+}
+
+const statesSchema = {
+	// eslint-disable-next-line @typescript-eslint/member-delimiter-style
```

Comment:
I don't think there's a compelling reason to disable the rule here instead of just doing what it says we should